styles.css

```diff
@@ -432,3 +437,632 @@ footer {
 .footer-socials li {
   list-style: none;
 }
+
+/* Desktop version Media Query */
+
+@media screen and (min-width: 992px) and (max-width: 1399px) {
```

Comment:
- [ ] Good job adding this media query and its width pixels. I think that `1399px` is a bit larger. I suggest you stay around `1100px` as the max.

Reply:
Can I ise `@media (min-width: 992px)` instead of setting it from `992px` to `1100px`?